integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/db/dbreport/DBMediatorUseTransaction.java

```diff
@@ -17,208 +17,121 @@
-        JDBC_DRIVER = automationContext.getConfigurationValue(XPathConstants.DATA_SOURCE_DRIVER_CLASS_NAME);
-        serverConfigurationManager = new ServerConfigurationManager(context);
-        copyJDBCDriverToClassPath();
         super.init();
-        updatedSynapseContent = updateSynapseConfiguration();
-        handlingMysqlDB();
-        updateESBConfiguration(updatedSynapseContent);
+        HttpRequestUtil.doPost(new URL(getApiInvocationURL(API_URL + INIT_CONTEXT_1)), "");
+        HttpRequestUtil.doPost(new URL(getApiInvocationURL(API_URL + INIT_CONTEXT_2)), "");
     }
 
+    /**
+     * Test with UseTransaction flag. The transaction mediator is used to handle transactional behaviour.
+     * The commit operation is checked.
+     * @throws Exception
+     */
     @SetEnvironment(executionEnvironments = {ExecutionEnvironment.STANDALONE})
     @Test(groups = "wso2.esb", description = "Test UseTransaction option .Use in conjunction with Transaction mediator "
     )
-    public void testDBmediatorSuccessCase() throws AxisFault, SQLException {
-        int IBMcountDB1, IBMcountDB2;
-
-        IBMcountDB1 = getDatabaseResultsForDB1();
-        assertEquals(IBMcountDB1, 1, "Fault, invalid response");
-        IBMcountDB2 = getDatabaseResultsForDB2();
-        assertEquals(IBMcountDB2, 0, "Fault, invalid response");
-        axis2Client.sendSimpleStockQuoteRequest(getMainSequenceURL(), null, "IBM");
-        IBMcountDB1 = getDatabaseResultsForDB1();
-        assertEquals(IBMcountDB1, 0, "Fault, Record Not Deleted from Database1");
-        IBMcountDB2 = getDatabaseResultsForDB2();
-        assertEquals(IBMcountDB2, 1, "Fault, Record Not Inserted to Database2");
+    public void testDBmediatorSuccessCase() throws Exception {
+        String ibmStringDB1, ibmStringDB2;
+        ibmStringDB1 = getDatabaseResultsForDB1();
+        assertTrue(ibmStringDB1.contains("IBM"), "Fault, invalid response");
+        ibmStringDB2 = getDatabaseResultsForDB2();
+        assertFalse(ibmStringDB2.contains("IBM"), "Fault, invalid response");
+        HttpRequestUtil.doPost(new URL(getApiInvocationURL(API_URL + COMMIT_CONTEXT + "?nameEntry=IBM")), "");
+        ibmStringDB1 = getDatabaseResultsForDB1();
+        assertFalse(ibmStringDB1.contains("IBM"), "Fault, Record Not Deleted from Database1");
+        ibmStringDB2 = getDatabaseResultsForDB2();
+        assertTrue(ibmStringDB2.contains("IBM"), "Fault, Record Not Inserted to Database2");
 
     }
+
+    /**
+     * Test with UseTransaction flag. The transaction mediator is used to handle transactional behaviour.
+     * The rollback operation is checked.
+     * @throws Exception
+     */
     @SetEnvironment(executionEnvironments = {ExecutionEnvironment.STANDALONE})
     /*JIRA issue: https://wso2.org/jira/browse/ESBJAVA-1553*/
     @Test(groups = "wso2.esb", enabled = false, description = "Test UseTransaction option ." +
```

Comment:
Looks like we have disabled test case (enabled=false).. Is it intentional?

Reply:
No it's not. Enabled the testcase.